app/tests/spec/views/complete_sign_up.js

```diff
@@ -583,10 +575,6 @@ define(function (require, exports, module) {
             }
           ));
         });
-
-        it('shows the success message', function () {
-          assert.isTrue(view.isSuccessVisible());
```

Comment:
Worth asserting the contents of the success element instead? (I realise you assert that in the mixin tests too, so no worries if it seems a bit belt+braces)